From c2c147532590daccb42956c5ca799544b9113ab6 Mon Sep 17 00:00:00 2001
From: Colin Patrick Mccabe <cmccabe@cloudera.com>
Date: Thu, 16 Jun 2016 16:45:41 -0700
Subject: [PATCH 1821/2748] HADOOP-13284. FileSystemStorageStatistics must not
 attempt to read non-existent rack-aware read
 stats in branch-2.8 (Mingliang Liu via cmccabe)

(cherry picked from commit b07cd9c6a994bfcd993bc58f392175255ebf7341)

Change-Id: I144470b203bc1b28ce5c05c6ed56acfcfdaeb6a3
---
 .../hadoop/fs/FileSystemStorageStatistics.java     |    4 ----
 1 file changed, 4 deletions(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystemStorageStatistics.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystemStorageStatistics.java
index 171b927..c0b4409 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystemStorageStatistics.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/FileSystemStorageStatistics.java
@@ -42,10 +42,6 @@
       "readOps",
       "largeReadOps",
       "writeOps",
-      "bytesReadLocalHost",
-      "bytesReadDistanceOfOneOrTwo",
-      "bytesReadDistanceOfThreeOrFour",
-      "bytesReadDistanceOfFiveOrLarger"
   };
 
   private static class LongStatisticIterator
-- 
1.7.9.5

